src/models/otp.ts

```diff
@@ -88,9 +88,21 @@ export class OTPEntry implements OTPEntryInterface {
           hash: string;
           index: number;
         },
-    encryption?: EncryptionInterface
+    encryption?: EncryptionInterface | Map<string, EncryptionInterface>
```

Comment:
This should just be a `Map<string, EncryptionInterface>`, the only place we need to have a case for `EncryptionInterface` is before migration logic runs.